docs/source/sci_api_tutorial/dipole.rst

```diff
@@ -0,0 +1,153 @@
+==============================================
+Bond Dipole
+==============================================
+
+.. note::
+
+    | **This bond_dipole calculation requires Multiwfn installed**
+
+.. note::
+
+    Science API is a special concept in EnzyHTP. They stand for those top-layer APIs
+    the are supposed to be used in assembling the `workflow <https://enzyhtp-doc.readthedocs.io/en/latest/sci_api_tutorial/how_to_assemble.html#find-the-science-api-that-directly-gives-what-you-need>`_.
+
+**You will find a detailed tutorial on how to use this Science API.**
+
+Input/Output
+=========================================================================================
+
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    .. image:: ../../figures/dipole_bond.svg
+        :width: 100%
+        :alt: dipole_bond
+
+Input
+-----------------------------------------------------------------------------------------
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    The parameters of the bond_dipole API contain the following option
+
+**Input**: The ``ele_stru``, ``atom_1``, ``atom_2`` and  are required input
+
+    - ``ele_stru``:
+        Electronic structure of the QM (Quantum Mechanics) region containing the target bond.
+        .. admonition:: How to obtain
+            
+            | A ElectronicStrutcure() object can be obtained by these `APIs <obtaining_ele_stru.html>`_.
+
+    - ``atom_1`` and ``atom_2``:
+        Atoms define the target bond within the region.
+        .. admonition:: How to obtain
+            
+            | A atom_1 and atom_2 `Atom()` object can be obtained by these `APIs <obtaining_Atom.html>`_ with `create_region_from_selection_pattern()`. 
+
+    - ``method``:
+        Keyword specifying the algorithm & software for dipole calculation (default method is "LMO-Multiwfn"). Check the Multiwfn manual for the alternative methods. Utilizes the Lu-Chen method for dipole calculation, where 2-center LMO dipole is defined by the deviation of the electronic mass center relative to the bond center.
+        .. admonition:: What's the option
+            
+            | The current option is "LMO-Multiwfn"
+
+    - ``work_dir``:
+        Working directory containing all the files in the calculation process.
+
+    - ``keep_in_file``:
+        Whether to keep the input file of the calculation.
+
+    - ``cluster_job_config``:
```

Comment:
We would like to add a note in the "Input" section that tell user they can choose to run it via ARMer by specifying cluster_job_config. And link it to the ARMer page, and the api-config-dict page, repectively.